src/Microsoft.ML.Core/Data/RoleMappedSchema.cs

```diff
@@ -98,68 +108,109 @@ public sealed class RoleMappedSchema
         private const string NameString = "Name";
         private const string FeatureContributionsString = "FeatureContributions";
 
+        /// <summary>
+        /// Instances of this are the keys of a <see cref="RoleMappedSchema"/>. This class also some holds important
```

Comment:
>some holds [](start = 96, length = 10)

are you sure it's correct order?

Reply:
Um no. Sorry. :) Me speak english pretty one day.

---
In reply to: [200456054](https://github.com/dotnet/machinelearning/pull/496#discussion_r200456054) [](ancestors = 200456054)